### PR TITLE
Added new verbiage

### PR DESCRIPTION
**Pull request by mkerns1**
## Summary

- Updated expenses questions verbiage

## Related issue(s)

- department-of-veterans-affairs/va.gov-team#71795

## Acceptance criteria
-   Update verbiage for Dependent Expenses

### CODE REVIEW

File: src/applications/ezr/components/FormDescriptions/index.jsx
Comment (longmd) [inline src/applications/ezr/components/FormDescriptions/index.jsx:1]: Can you please rename this file to match the name of the const and export similar to `ContactInfoDescription`? We need to also create a unit test to ensure rendering.

File: src/applications/ezr/config/chapters/householdInformation/dependentEducationExpenses.js [hunk 1]
```diff
@@ -8,6 +8,7 @@ import { replaceStrValues } from '../../../utils/helpers/general';
 import { validateCurrency } from '../../../utils/validation';
 import { LAST_YEAR } from '../../../utils/constants';
 import content from '../../../locales/en/content.json';
+import { DependentEducationExpensesDescription } from '../../../components/FormDescriptions';
```
Comment (longmd) [inline src/applications/ezr/config/chapters/householdInformation/dependentEducationExpenses.js:11]: Update this import upon completing the adjustment for the file naming and export convention above.

**Review by longmd — CHANGES_REQUESTED** [review]

File: src/applications/ezr/components/FormDescriptions/DependentEducationExpensesDescription.jsx
Comment (longmd) [inline src/applications/ezr/components/FormDescriptions/DependentEducationExpensesDescription.jsx:1]: Can you add a unit test for this file to ensure we have test coverage for it? You can duplicate what was done for the contact info description.

**Review by longmd — DISMISSED** [review]

**Review by longmd — DISMISSED** [review]
LGTM. Thank you for the updates.

**Review by ataker — DISMISSED** [review]
